persistence-typed/src/main/scala/org/apache/pekko/persistence/typed/javadsl/CommandHandlerWithReply.scala

```diff
@@ -117,7 +116,7 @@ final class CommandHandlerWithReplyBuilder[Command, Event, State]() {
    * @return A new, mutable, CommandHandlerWithReplyBuilderByState
    */
   def forNullState(): CommandHandlerWithReplyBuilderByState[Command, Event, State, State] = {
-    val predicate: Predicate[State] = asJavaPredicate(s => Objects.isNull(s))
+    val predicate = ((s: State) => Objects.isNull(s)).asJava
```

Comment:
Hah, I see

Reply:
Btw, Why can this be ：
```scala
val predicate: Predicate[State] = (s: State) => Objects.isNull(s)
```

Reply:
> Btw, Why can this be ：
> 
> ```scala
> val predicate: Predicate[State] = (s: State) => Objects.isNull(s)
> ```

Nope, there aren't implicit conversions. `(s: State) => Objects.isNull(s)` is of type `Function1[State, Boolean]` and the .`asJava` turns that into a `Predicate[State]`).

Reply:
Yes, the type needs to be `Predicate[State]` (which is actually a Java `Functional` interface)